package.json

```diff
@@ -153,8 +153,7 @@
       "redux-saga/yield-effects": 2,
       "require-yield": 0,
       "import/no-webpack-loader-syntax": 0,
-      "import/first": 0,
-      "no-restricted-syntax": 0
```

Comment:
One question: does anyone iterate over generators with the `for ... of` loop. As far as I remember, Airbnb recommends against generators, this is why they don't care about the `for ... of` loop.